packages/terra-icon/CHANGELOG.md

```diff
@@ -3,11 +3,20 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated Person Door icon
+* Bumped `one-cerner-style-icons` to v1.30.1
+* Changed WDIO tests to use `huge` viewport in order to accomodate the new icons introduced in v1.30.1
 
 3.25.0 - (January 7, 2020)
 ------------------
 ### Changed
-* Updated eIFU text to be `Electronic Instructions for Use Icon`.
+* Updated eIFU text to be `Electronic Instructions for Use Icon`
+* Bumped `one-cerner-style-icons` to v1.29.2
+* Updated `terra-dev-site` WDIO test for IconAll
```

Comment:
The merge after the January 7th release must have shifted these into a released section. Can you move these into the unreleased section? The one-cerner-style-icons entry can be removed entirely since the newer version is listed above.